### ISSUE TITLE
DEV: Update bulk sending flow

### ISSUE THREAD

**Issue by yaelberger-commits** [cds-snc/notification-planning#741]
_Update bulk sending flow_

## Description

Update content through code in authenticated pages of GC Notify for the bulk sending flow to include new information and changes about sending using a spreadsheet and guidance, and error messages when something goes wrong

## Acceptance Criteria** (Definition of done) 

- [x] Adding spreadsheet guidance into bulk sending flow
- [x] updating error messages for using a spreadsheet
- [x] updating descriptions on "add recipient" page

* Definition of Done **does not** include fixing bug with indexed numbers don't align with rows #744

_To be refined through discussion with the team_

Given a user is going through the bulk sending flow with a spreadsheet, when they visit each of the various pages of the flow, or encounter an error, they will see the correct new information and error messages and descriptions in both EN or FR. 

## QA Steps

- [x] Check that the content in staging and prod matches the content in Figma
- [x] Check that each error scenario for the spreadsheets results in the correct error message in both EN and FR
- [x] Check using the different browsers (Chrome, Firefox, Safari)
- [x] A11y manual testing (screen readers)

**Comment by amazingphilippe** [comment]
Figma for translated content: https://www.figma.com/file/uerfGAwA8MqaZ65Ep8oeXl/Recipients-files-guidance---Guide-fichiers-de-destinataires?node-id=934%3A2947

Figma for a11y and dev annotations: https://www.figma.com/file/uerfGAwA8MqaZ65Ep8oeXl/Recipients-files-guidance---Guide-fichiers-de-destinataires?node-id=984%3A3052

**Comment by yaelberger-commits** [comment]
In Figma, look for the black back that says changes for devs in Notify App done in notification-admin

**Comment by amazingphilippe** [comment]
Noticed we were missing a few error states. [Documented in Figma to work on](https://www.figma.com/file/uerfGAwA8MqaZ65Ep8oeXl/Recipients-files-guidance---Guide-fichiers-de-destinataires?node-id=1265%3A4901)

**Comment by jzbahrai** [comment]
We are we waiting on content from this and then @amazingphilippe is required for comments.

**Comment by amazingphilippe** [comment]
Wrapping up this work today. Maybe write some scenarios with Ramya for testing this one with all the possible error messages possibilities.

**Comment by amazingphilippe** [comment]
A11y checks done with ARC toolkit and Axe. Found 1 bug, already fixed in another PR. Moving this to done.
